test/jdk/TEST.groups

```diff
@@ -585,6 +585,8 @@ jdk_core_manual_no_input = \
     javax/net/ssl/compatibility/HrrTest.java \
     javax/net/ssl/compatibility/SniTest.java \
     jdk/nio/zipfs/TestLocOffsetFromZip64EF.java
```

Comment:
Shouldn't we add here a backslash at the end of the line?

Reply:
Good catch! I will add it back.